assets/js/components/Banner/LearnMoreLink.tsx

```diff
@@ -29,18 +29,36 @@ import { __ } from '@wordpress/i18n';
  */
 import Link from '@/js/components/Link';
 
-export default function LearnMoreLink( {
+/**
+ * A "Learn More" link component, usually used to append "Learn more" with a
+ * specified URL to content like a paragraph.
+ *
+ * @since 1.153.0
+ */
+const LearnMoreLink: FC< {
```

Comment:
I would better prefer having interfaces added for props rather than converting all functional components to arrow functions with multilines signature.
interface LearnMoreLinkProps {
	href: string;
	className?: string;
	label?: string;
	external?: boolean;
	onClick?: () => void;
}

export function LearnMoreLink( props: LearnMoreLinkProps ) {
    const { ... } = props;
    ...
}
To me it is easier to read and understand.

Reply:
I actually prefer deconstructing the properties inline as we do now, but I don't have a strong opinion on that, and don't mind if we change to the `props` argument approach.